pysat/instruments/methods/testing.py

```diff
@@ -1,4 +1,51 @@
+import os
 import numpy as np
+import pandas as pds
+import pysat
+
+
+def list_files(tag=None, sat_id=None, data_path=None, format_str=None,
+               file_date_range=None, test_dates=None):
+    """Produce a fake list of files spanning a year
+
+    Parameters
+    ----------
+    tag : (str)
+        pysat instrument tag (default=None)
+    sat_id : (str)
+        pysat satellite ID tag (default=None)
+    data_path : (str)
+        pysat data path (default=None)
+    format_str : (str)
+        file format string (default=None)
+    file_date_range : (pds.date_range)
+        File date range (default=None)
```

Comment:
We do construct a default time range, 3 years, associated with inst.test_dates that should be noted in the doc string. I would also note that sat_id and tag should be used to access test_dates except for that fact that those keywords are currently overloaded.

Reply:
Can't seem to pass these through as currently set up.

Reply:
No, the list_files method definition is only executed once in pysat_testing*.py as
list_files = functools.partial(mm_test.list_files, test_dates=_test_dates)

It takes the _test_dates dictionary defined within each pysat.Instrument. It is only evaluated once then list_files runs each time after with the same input to test_dates.